### PR TITLE
FIX Check workflow is active

### PR DESCRIPTION
**Pull request by emteknetnz**
Issue https://github.com/silverstripe/.github/issues/320

silverstripe/recipe-core builds are [failing](https://github.com/silverstripe/recipe-core/actions/runs/11152360312/job/31653517024) because it stubbornly insists that that a tag-patch-release.yml workflow exists, even though it's no where to be found in the repo anymore - https://api.github.com/repos/silverstripe/recipe-core/actions/workflows

I've manually disabled in it https://github.com/silverstripe/recipe-core/actions/workflows/tag-patch-release.yml - this shows up in the API under 'state', so filter on that

### CODE REVIEW

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -171,7 +171,7 @@ jobs:
         options: --health-cmd="healthcheck.sh --connect --innodb_initialized" --health-interval=10s --health-timeout=5s --health-retries=10
 
     env:
-      artifacts_name: ${{ matrix.name }}
+      artifacts_name: ${{ matrix.artifacts_name }}
```
Comment (ChloeSartorelli) [inline .github/workflows/ci.yml]: What's this change about? Doesn't seem related to what you've mentioned in the PR description.

**Review by GuySartorelli — CHANGES_REQUESTED** [review]

File: .github/workflows/ci.yml (hunk 1 above)
Comment (emteknetnz) [reply .github/workflows/ci.yml]: Reverted

**Review by GuySartorelli — APPROVED** [review]
LGTM
